lib/puppet/provider/user/aix.rb

```diff
@@ -216,7 +216,9 @@ def expiry_from_attr(value)
 
   def open_security_passwd
     # helper method for tests
-    File.open("/etc/security/passwd", 'r')
+    # AIX reference indicates this file is ASCII
+    # https://www.ibm.com/support/knowledgecenter/en/ssw_aix_72/com.ibm.aix.files/passwd_security.htm
+    Puppet::FileSystem.open("/etc/security/passwd", nil, "r:ASCII")
```

Comment:
A regular spec would be of dubious value I think - would only serve as double book-keeping.

I think there are 2 existing acceptances which probably cover this (at least indirectly) as they set a password:
* https://github.com/puppetlabs/puppet/blob/ebd96213cab43bb2a8071b7ac0206c3ed0be8e58/acceptance/tests/resource/user/should_modify_while_managing_home.rb
* https://github.com/puppetlabs/puppet/blob/ebd96213cab43bb2a8071b7ac0206c3ed0be8e58/acceptance/tests/resource/user/should_modify_when_not_managing_home.rb